third_party/terraform/resources/resource_google_project_service.go

```diff
@@ -82,10 +82,11 @@ func resourceGoogleProjectService() *schema.Resource {
 				ValidateFunc: StringNotInSlice(append(ignoredProjectServices, bannedProjectServices...), false),
 			},
 			"project": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Computed: true,
-				ForceNew: true,
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: compareResourceNames,
```

Comment:
`compareSelfLinkOrResourceName` is more typical- any specific reason you chose this one over that?

Reply:
compareResourceNames is simpler under the hood. compareSelfLinkOrResourceName has a lot of complexity that I feel I didn't need. I tried to use it originally but ran into some issue. I try to switch to compareSelfLinkOrResourceName if you think that's the better practice

Reply:
All good! just stood out that this was a non-standard one.